CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Changelog
 
-## master / unreleased
+## 1.1.0 / 2020-05-??
+
+This release brings the usual mix of bugfixes and improvements. The biggest change is that WAL support for chunks is now considered to be production-ready!
+
+Please make sure to review renamed metrics, and update your dashboards and alerts accordingly.
 
 * [CHANGE] Added v1 API routes documented in #2327. #2372
   * Added `-http.alertmanager-http-prefix` flag which allows the configuration of the path where the Alertmanager API and UI can be reached. The default is set to `/alertmanager`.
   * Added `-http.prometheus-http-prefix` flag which allows the configuration of the path where the Prometheus API and UI can be reached. The default is set to `/prometheus`.
   * Updated the index hosted at the root prefix to point to the updated routes.
   * Legacy routes hardcoded with the `/api/prom` prefix now respect the `-http.prefix` flag.
-* [CHANGE] The metrics `cortex_distributor_ingester_appends_total` and `distributor_ingester_append_failures_total` now includes a `type` label to differentiate between `samples` and `metadata`. #2336
+* [CHANGE] The metrics `cortex_distributor_ingester_appends_total` and `distributor_ingester_append_failures_total` now include a `type` label to differentiate between `samples` and `metadata`. #2336
 * [CHANGE] The metrics for number of chunks and bytes flushed to the chunk store are renamed from: #2463
```

Comment:
Can you add a note that the metrics counted here are _after_ deduplication while the metrics before were pre-deduplication?